features/brexit_check.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Get Ready for Brexit Check
+
+  Background:
+    Given I am testing through the full stack
+    And I force a varnish cache miss
+
+  @high
+  Scenario: Brexit checker shows results
+    Given I am a UK national
```

Comment:
This was surprising, as it seems to exist purely for the purpose of a subsequent step. I had to read as I couldn't guess what 'being a UK national' translates to on GOV.UK. Could we just assume the step that answers the question is going to inject some sensible data?

Reply:
Updated to remove these given clauses.